tools/run_tests/run_tests_matrix.py

```diff
@@ -130,6 +130,15 @@ def _create_test_jobs(extra_args=[], inner_jobs=_DEFAULT_INNER_JOBS):
                              extra_args=extra_args,
                              inner_jobs=inner_jobs)
 
+  for compiler in ['python3.4', 'python3.5', 'python3.6', 'pypy']:
```

Comment:
portability test runs with --build_only and it does not run any tests (I understand that's a bit annoying for dynamic languages like python).

Btw, it turns out that python tests now run much faster than they used to (it used to be up to 30mins, now it seems to be more like 4mins). So if there's some added value in running full tests for more versions of python (e.g. 2.7, 3.4 - the oldest python3 and 3.6 the newest python3 atm), it might be just fine.